sites/skeleton.dev/src/content/docs/get-started/installation/nextjs.mdx

```diff
@@ -47,6 +47,7 @@ import ProcessStep from '@components/docs/ProcessStep.astro';
         @import '@skeletonlabs/skeleton';
         @import '@skeletonlabs/skeleton/optional/presets';
         @import '@skeletonlabs/skeleton/themes/cerberus';
+        @import '@skeletonlabs/skeleton-react';
```

Comment:
Let's keep the import orders:

- Core (required)
- Framework package (required)
- Theme (optional or swappable)
- Presets (optional)

As the first couple are pretty important.

This goes for all install guides.